python_modules/dagster/dagster/_core/definitions/asset_selection.py

```diff
@@ -182,14 +181,10 @@ def checks_for_assets(*assets_defs: AssetsDefinition) -> "AssetChecksForAssetKey
 
     @public
     @staticmethod
-    def checks(*asset_checks: AssetChecksDefinition) -> "AssetCheckKeysSelection":
+    def checks(*assets_defs: AssetsDefinition) -> "AssetCheckKeysSelection":
```

Comment:
Perhaps not ideal but prefer to leave this as `AssetsDefinition` for now since this method also gets used internally, and internally we just work with `AssetsDefinition` and not `AssetChecksDefinition`. This won't cause any user problems bc it's a subclass.